README.md

```diff
@@ -92,7 +92,7 @@ Example Playbooks
       roles:
 
       - role: ome.postgresql
-        postgresql_install_server: True
+        postgresql_version: "10"
```

Comment:
This isn't a change, it should've been added after https://github.com/ome/ansible-role-postgresql/releases/tag/4.0.0 was released

Reply:
Should the version not be set too in the molecule tests?

Reply:
It's included in the `host_vars` in molecule.yml to allow different versions of postgres to be tested.